monitoring/README.md

```diff
@@ -92,47 +174,124 @@ c) Upload the dashboard JSON files from your local `./grafana` sub-directory int
 - `NS`: name of the KubeStellar space namespace, e.g., *wds1-system*, *wds2-system*, *its1-system*, etc.
 - `APIService`: name of the service for a KubeStellar space APIserver, e.g., *wds1*, *wds2*, *vcluster*, etc.
+- `Instance`: hostname and port number for the target KubeStellar space APIserver, e.g., *10.244.0.18:9444*
 
     <img src="images/apiserver-apf-monitoring.png" width="60%" height="80%" title="apiserver-apf-monitoring.png"> 
 
-Note: use the same combinations of `NS` and `APIService` parameters described in `step 5-a` to monitor the APIserver in the WDS space, ITS space or the hosting cluster. 
+**Note**: use the same combinations of `NS` and `APIService` parameters described in `step 5-a` to monitor the APIserver in the WDS space, ITS space, the hosting or WEC clusters. 
 
 
 c) KubeStellar Controllers Monitoring: input into the following text box and drop-down menu. 
 
-- `SpaceNS`: name of the KubeStellar space namespace, e.g., *wds1-system*, *wds2-system*, etc.
+- `NS`: name of the KubeStellar space namespace, e.g., *wds1-system*, *wds2-system*, etc.
+
+- `KSController`: select the KubeStellar controller name. Available options from the drop-down menu: *kubestellar-controller-manager-metrics-service*, *ks-transport-controller*, *status-addon-controller* and *status-agent-controller*. 
 
-- `KSController`: select the KubeStellar controller name. Available options from the drop-down menu: *kubestellar-controller-manager-metrics-service*, *ks-transport-controller* and *status-addon-controller*. 
+- `Instance`: hostname and port number for the target KubeStellar space controller, e.g., *10.244.0.71:8090*
 
     <img src="images/ks-controllers-monitoring.png" width="60%" height="80%" title="ks-controllers-monitoring">
 
-To monitor the KubeStellar controllers in a WDS space (e.g., wds1) use the following parameters: 
-- `SpaceNS: wds1-system` and `KSController: kubestellar-controller-manager-metrics-service`
-- `SpaceNS: wds1-system` and `KSController: ks-transport-controller`
+To monitor the KubeStellar controllers in a WDS space (e.g., wds1) or ITS space (e.g., its1) use the following combination of `NS/APIService` parameters: 
+- `NS: wds1-system` and `KSController: kubestellar-controller-manager-metrics-service`
+- `NS: wds1-system` and `KSController: ks-transport-controller`
+
+Likewise, to monitor the KubeStellar controller in a ITS space and WEC (e.g., its1): `NS: its1-system` and `KSController: status-addon-controller` & `NS: open-cluster-management-agent-addon` and `KSController: status-agent-controller`.
+
+Note: 
+- Use the following commands to obtain the Ip address of the KubeStellar controllers in the WDS (e.g., `wds1`) and ITS (e.g., `its1-system`) spaces and WEC (e.g.,`cluster1`):
+
+  ```bash
+  kubectl --context kind-kubeflex -n wds1-system get Endpoints | egrep ".*controller"
+  ```
+  ```bash
+  kubectl --context kind-kubeflex -n its1-system get Endpoints | egrep ".*controller"
```

Comment:
I am testing the latest rev of this PR (commit b249c02a8e45eadcfbf088d288f91cb48cb114fb). I get "no data" in two of the six panels in the status-addon-controller case.

<img width="880" alt="image" src="https://github.com/user-attachments/assets/1c2c5b7b-efc9-4017-939a-1a25c250398b">

Reply:
This is correct - this is relative to the missing metrics issues from our controllers that I reported in the past. Also, see this issue: https://github.com/kubestellar/kubestellar/issues/2158

Reply:
I think it would be good if this document tells the reader not to expect data in those panels.